bootstrap.d/sys-apps.yml

```diff
@@ -555,19 +555,11 @@ packages:
     architecture: '@OPTION:arch@'
     source:
       subdir: ports
-      git: 'https://git.savannah.gnu.org/git/man-db.git'
-      tag: '2.9.4'
-      version: '2.9.4'
-      tools_required:
-        - host-autoconf-v2.69
-        - host-automake-v1.15
-        - host-pkg-config
-        - host-libtool
-      regenerate:
-        - args: ['./bootstrap']
-        - args: ['cp',
-            '@BUILD_ROOT@/tools/host-automake-v1.15/share/automake-1.15/config.sub',
-            '@THIS_SOURCE_DIR@/build-aux/']
+      url: 'https://download-mirror.savannah.gnu.org/releases/man-db/man-db-2.12.0.tar.xz'
+      format: 'tar.xz'
+      checksum: blake2b:a6258c1fc9cd81be5d288298660f5b9bda22d726ef98dd5c0a1998809a32391f7244be6897ee8e03483e705a426d6a59d442de3dc2e84f45500daa825ac639db
+      extract_path: 'man-db-2.12.0'
+      version: '2.12.0'
```

Comment:
Likewise.

Reply:
Update dropped due to autoreconf problems.